gensim/corpora/textcorpus.py

```diff
@@ -177,12 +97,12 @@ class TextCorpus(interfaces.CorpusABC):
 
     The default preprocessing consists of:
 
-    #. :func:`~gensim.corpora.textcorpus.lower_to_unicode` - lowercase and convert to unicode (assumes utf8 encoding)
+    #. :func:`~gensim.parsing.preprocessing.lower_to_unicode` - lowercase and convert to unicode (assumes utf8 encoding)
     #. :func:`~gensim.utils.deaccent`- deaccent (asciifolding)
-    #. :func:`~gensim.corpora.textcorpus.strip_multiple_whitespaces` - collapse multiple whitespaces into a single one
+    #. :func:`~gensim.parsing.preprocessing.strip_multiple_whitespaces`- collapse multiple whitespaces into a single one
```

Comment:
Is this change intentional?

Reply:
Yes. As the TextCorpus class now using the `strip_multiple_whitespaces()` function from the `preprocessing` module, I have changed that line in documentation also.

Reply:
I mean the deleted space.

Reply:
sorry, I got it wrong.

Yes, I had changed it intentionally. It was giving PEP8 error -
`E501 line too long (121 > 120 characters)`

Should I keep that space?

Reply:
Of course.

Reply:
Done the fixing as per suggestion.